manifests/f/flxzt/rnote/0.7.1/flxzt.rnote.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: flxzt.rnote
+PackageVersion: 0.7.1
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/flxzt/rnote/releases/download/v0.7.1/rnote-win-installer-0.7.1-x86_64.exe
+  InstallerSha256: 9F917FCD3F3027581F937825CBC576B908504D6BF0B8B667C941B27698A0C40F
+  UpgradeBehavior: uninstallPrevious
```

```suggestion
  UpgradeBehavior: uninstallPrevious
  ProductCode: '{14CA6E6F-6384-467B-B139-D6305C0F4431}_is1'
```
Additional metadata for upgrade matching